### PR TITLE
Add 'Graphics stack' page from wiki

### PR DESCRIPTION
**Pull request by pyhalov**
This PR contains most important parts from https://wiki.openindiana.org/display/oi/Graphic+stack besides https://wiki.openindiana.org/display/oi/Intel+KMS+driver 'Testing' section. I consider this section should be moved to HCL.

### CODE REVIEW

**Comment by pyhalov** [comment]
Generated pages: http://buildzone.oi-build.r61.net/site/dev/graphics-stack/

**Comment by pyhalov** [comment]
@alarcher @Mno-hime 
Will you take a look?

File: docs/dev/graphics-stack.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+All the Xorg components are build in oi-userland and located in the x11 directory:
+
+<https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/components/x11>
+
+A bogus component is provided to rebuild the X11 gate:
+<https://github.com/OpenIndiana/oi-userland/tree/oi/hipster/components/x11/x11-gate>
+
+The upstream Solaris gate is located at:
+
+<https://github.com/oracle/solaris-xorg/>
```
Comment (Mno-hime) [inline docs/dev/graphics-stack.md]: Certainly dead? The page says "This repository has been archived by the owner. It is now read-only."

File: docs/dev/graphics-stack.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+* PineView-D (Atom D400/D500 series)
+* Intel(R) HD Graphics,
+* Intel(R) Iris(TM) Graphics,
+* Intel(R) Iris(TM) Pro Graphics.
+
+Open source kernel driver(s) by Oracle:
+
+<https://github.com/oracle/solaris-xorg/tree/master/open-src/kernel>
+
+Relevant kernel code is located in drm and i915 subfolders.
```
Comment (Mno-hime) [inline docs/dev/graphics-stack.md]: Irrelevant because it's dead and we have gfx-drm gate?

File: docs/dev/graphics-stack.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+
+For now, UXA mode is set by default.
+
+#### Sandy Bridge GT1
+
+A hardware bug requires implementing a workaround in the Intel ringbuffer implementation, occasional 1-2 second hangs may occur.
+
+#### Gen >= 8
+
+Generation 8 specifics in Intel ringbuffer are not implemented yet.
```
Comment (Mno-hime) [inline docs/dev/graphics-stack.md]: Better explicitly say what is hidden in the table below: "Gen >=8 (Broadwell and newer) not supported at all."

File: docs/dev/graphics-stack.md [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+        7th | HASWELL M      | Haswell              | 0x0D06  |CRW GT1 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0D16  |CRW GT2 mobile         |              |
+        7th | HASWELL M      | Haswell              | 0x0D26  |CRW GT3 mobile         |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f30|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f31|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f32|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0f33|                      |              |
+        7th | VALLEY VIEW M  | Valley View (Bay Trail) | 0x0157|                      |              |
+        7th | VALLEY VIEW D  | Valley View (Bay Trail) | 0x0155|                      |              |
+        8th | BROADWELL GT12M| Broadwell            | 0x1602  | GT1 ULT               |Gen >=8 **NOT** supported at all.|
```
Comment (Mno-hime) [inline docs/dev/graphics-stack.md]: I guess remove gen 8 and newer completely? It does not seem to suit anything.

Comment (pyhalov) [reply docs/dev/graphics-stack.md]: It's useful at least for one purpose - check device id and see that it's completely unsupported :)

**Pull request merged**